src/plugins/controls/public/control_types/range_slider/range_slider_embeddable.tsx

```diff
@@ -235,58 +217,99 @@ export class RangeSliderEmbeddable extends Embeddable<RangeSliderEmbeddableInput
       }
     }
 
+    const { min, max } = await this.fetchMinMax({
+      dataView,
+      field,
+      filters,
+      query,
+    });
+
+    this.updateComponentState({
+      min: `${min ?? ''}`,
+      max: `${max ?? ''}`,
+    });
```

Comment:
There are problems like that in the `ecommerce` dataset for the Options List controls too. Using different nested and array types in ES is really not an ideal solution, and it will be tough for us to fix all the strangeness that comes about when using mappings like that from the Kibana side.